cla.html

```diff
@@ -17,15 +17,101 @@ <h1 class="h2">Contributor License Agreements</h2>
         <div class="wrapper my-6 py-4">
             <div class="col-md-10 col-lg-7 mx-auto">
                 <h3 class="h4 font-weight-400 mb-4">
-                  Microsoft projects have a Contributor License Agreement (CLA). The Microsoft 
-                  CLA only needs to be agreed to once. We aim to make participating as easy
-                  and efficient as possible, and thank you for your contributions to the
-                  open source ecosystem.
+                    We appreciate community contributions to Microsoft repositories. By
+                    signing a contributor license agreement, we ensure that the
+                    community is free to use your contributions.
                 </h3>
             </div>
         </div>
     </div>
 
-    <!-- TODO: additional content -->
+
+    <div class="wrapper my-6 py-4">
+        <div class="col-md-10 col-lg-7 mx-auto longform">
+            <p class="p-lg">
+                TBD.
+            </p>
+
+            <br />
+            <br />
+
+            <div class="pb-6">
+                <h3 class="h3">Review the CLA document</h3>
+                <p>
+                    Download the <a href="/pdf/microsoft-contribution-license-agreement.pdf" target="_new">Microsoft Contributor License Agreement (CLA) PDF</a> document if you need.
+                </p>
+            </div>
+            <div class="pb-6">
+                <h3 class="h3">Signing the CLA</h3>
+                <p>
+                    The Microsoft CLA experience is integrated natively into GitHub and its pull request
+                    experience. Here's how it works the first time you contribute to a Microsoft open source
+                    project:
+                </p>
+                 <ul>
+                    <li>
+                        When you open a pull request, the <code>license/cla</code> check is run by our 
+                        bot.
+                    </li>
+                    <li>
+                        If you've signed the CLA before, the status check goes green, and you're done, your
+                        pull request is ready to be reviewed and merged.
```

Comment:
"If you've signed the CLA before, the status check goes green, and you're **done. Your** pull request is ready to be reviewed and merged."